### PR TITLE
HWKAGENT-72 be able to select the metrics from prometheus endpoints to store

### PR DESCRIPTION
**Pull request by jmazzitelli**

### CODE REVIEW

File: hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/dynamicprotocol/DynamicEndpointService.java [hunk 1]
```diff
@@ -33,16 +36,17 @@
     private final MonitoredEndpoint<DynamicEndpointConfiguration> endpoint;
     private final String feedId;
     private final HawkularWildFlyAgentContext hawkularStorage;
+    private final Collection<Name> metrics; // if not empty, these name the metrics this service should monitor
```
Comment (jshaughn) [inline hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/dynamicprotocol/DynamicEndpointService.java:39]: perhaps this should be pattern based with regex?

Comment (jmazzitelli) [inline hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/dynamicprotocol/DynamicEndpointService.java:39]: yes, I like that. OK, I'll make that change

**Comment by jshaughn** [comment]
Other than the regex comment, LGTM.

File: hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/dynamicprotocol/DynamicEndpointService.java (hunk 1 above)
Comment (jmazzitelli) [inline hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/dynamicprotocol/DynamicEndpointService.java:39]: last comment adds the regex feature. i'll merge soon

**Pull request merged**